pkg/ui/README.md

```diff
@@ -16,9 +16,9 @@ you'll be able to access the UI at <http://localhost:8080>.
 Our UI is compiled using a collection of tools that depends on
 [Node.js](https://nodejs.org/) and are managed with
 [Yarn](https://yarnpkg.com), a package manager that offers more deterministic
-package installation than NPM. NodeJS 6.x and Yarn 0.22.0 are known to work.
-[Chrome](https://www.google.com/chrome/), Google's internet browser. Unit tests
-are run using Chrome's "Headless" mode.
+package installation than NPM. NodeJS 8.4.x and Yarn 1.3.x are known to work.
```

Comment:
build/builder/Dockerfile

Reply:
I would drop version numbers in documentation. Keeping it at `>=6` in `package.json` is important, thought because that indicates support for the latest LTS version of Node.js, e.g. you don't want to drop that one.

Reply:
Unlike internal dep bumps, I don't think we want to push required versions on externally-installed build tools unless we've got a compelling reason for the change.  Also we don't want to change the builder image until we bump the required version, otherwise there's a good chance we'll accidentally push the required version later without noticing it (consider the accidental upgrade to go 1.10 as an example).

There may be a reasonable argument for `yarn` even if not for `node`.

In any case, we'll want to send an alert to the mailing list to make sure people are warned ahead of a build change requiring a manual upgrade.